tests/tensor_search/integ_tests/test_dict_score_modifiers.py

```diff
@@ -21,7 +21,6 @@ def setUpClass(cls) -> None:
         # UNSTRUCTURED indexes
         unstructured_default_text_index = cls.unstructured_marqo_index_request(
             model=Model(name='random/small'),
-            marqo_version="2.9.0"
```

Comment:
why do we remove the marqo_version here? Is this test case for old score modifiers?

Reply:
I think having that there was an oversight. unstructured_default_text_index should always be the latest version. @RaynorChavez Is there anything wrong with me removing this one line?

Reply:
Yes @vicilliar, we can safely remove this. This was an undeleted holdover from a previous version of the test. marqo_version by default is set to be the latest version in `unstructured_default_text_index`.

Reply:
Awesome, thanks!